### PR TITLE
COST-1575: Update OCP-on-Cloud calculations for distribution type (Trino)

### PR DESCRIPTION
**Pull request by myersCody**
TODO: Smoke tests

### CODE REVIEW

**Comment by myersCody** [comment]
Smoke Tests:
```
ERROR lib64/python3.6/site-packages/iqe_cost_management/tests/rest_api/v1/test_tagging.py::test_api_ocp_on_azure_tag_filtered_project_false - wait_for.TimedOutError: Could not do 'function check_ingest_complete()' at /iqe_venv/lib64/py...
ERROR lib64/python3.6/site-packages/iqe_cost_management/tests/rest_api/v1/test_tagging.py::test_api_ocp_on_azure_tagging_report_filter_match_date_range - wait_for.TimedOutError: Could not do 'function check_ingest_complete()' at /iqe_v...
=========================================================================== 12 failed, 5189 passed, 201 skipped, 638 deselected, 635 errors in 6351.82s (1:45:51) ============================================================================
There is a test failing in your suite. Debug it running:
		curl -L https://url.corp.redhat.com/iqe-sh > iqe.sh && chmod +x iqe.sh
		ENV_FOR_DYNACONF=local IQE_IMAGE=docker-registry.upshift.redhat.com/insights-qe/iqe-tests:cost_management IQE_TESTS_LOCAL_CONF_PATH=/iqe_conf ./iqe.sh tests plugin cost_management --pdb
(koku) bash-3.2$
```
Don't think these are related to my changes, but gonna look into it more tomorrow.

File: koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py [hunk 1]
```diff
@@ -77,6 +78,7 @@ def update_aws_summary_tables(self, openshift_provider_uuid, aws_provider_uuid,
                     current_ocp_report_period_id,
                     current_aws_bill_id,
                     markup_value,
+                    distribution,
```
Comment (adberglund) [inline koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py:81]: In this case, we want to check the distribution on the associated OpenShift cluster's cost model, *not* the cost model for the cloud source. We still get the markup from the cloud source, though.

**Review by adberglund — CHANGES_REQUESTED** [review]
Just a little change and this should be 👌

**Review by adberglund — APPROVED** [review]

**Pull request merged**